### ISSUE TITLE
Websocket typing

### ISSUE THREAD

**Issue by bfischer1121** [trycourier/courier-react#124]
Hi,

I believe the typings [here](https://github.com/trycourier/courier-react/blob/main/packages/react-provider/src/transports/types.ts) for websocket messages may be incorrect...

Here's an example with the typings and a payload we inspected...
```typescript
export interface ITextBlock {
  type: 'text'
  text: string
}

export interface IActionBlock {
  type: 'action'
  text: string
  url: string
}

export interface ICourierMessage {
  event?: string
  body?: string | React.ReactElement
  blocks?: Array<ITextBlock | IActionBlock>
  icon?: string | false
  title?: string | React.ReactElement
  data?: {
    clickAction?: string
    clickTrackingId?: string
    readTrackingId?: string
    deliverTrackingId?: string
  }
  brand?: any // dropped for simplicity in example
}

export const courierMessageEvent: ICourierMessage = {
  blocks: [],
  title: '...',
  data: {
    brandId: '...',
    trackingIds: {
      clickTrackingId: '...',
      deliverTrackingId: '...',
      readTrackingId: '...',
      unreadTrackingId: '...'
    }
  },
  event: '...'
}
```

**Comment by rileylnapier** [comment]
Hi there, we will take a look and get back to you.  Thanks for making an issue!

**Comment by rileylnapier** [comment]
Is it possible to share some example code of how you are implementing your own handler?

**Comment by rileylnapier** [comment]
closing this for now, if there is still an issue please reopen or make a new ticket
